chain/bitcoind_conn.go

```diff
@@ -16,6 +16,28 @@ import (
 	"github.com/lightninglabs/gozmq"
 )
 
+const (
+	// rawBlockZMQCommand is the command used to receive raw block
+	// notifications from bitcoind through ZMQ.
+	rawBlockZMQCommand = "rawblock"
+
+	// rawTxZMQCommand is the command used to receive raw transaction
+	// notifications from bitcoind through ZMQ.
+	rawTxZMQCommand = "rawtx"
+
+	// maxRawBlockSize is the maximum size for a raw block received from
+	// bitcoind through ZMQ.
+	maxRawBlockSize = 4e6
+
+	// maxRawTxSize is the maximum size for a raw standard transaction
+	// received from bitcoind through ZMQ.
+	maxRawTxSize = 4e5
```

Comment:
is there any harm in just using another 4MB? a malicious miner could sever bitcoind communication simultaneously if they violate policy, i would vote for safety here.

Reply:
> is there any harm in just using another 4MB?

Not at all.

> a malicious miner could sever bitcoind communication simultaneously if they violate policy, i would vote for safety here.

Seems like we'd just be unable to deserialize the transaction, which would only be crucial if the transaction was somehow relevant to us (unlikely?). Definitely an easy way to mitigate such harm by just increasing this to 4MB though, fixed.